check-links.sh

```diff
@@ -9,17 +9,18 @@ function get_tag_for_latest(){
     REPOSITORY="linaroits/linkcheck"
     TARGET_TAG="latest"
     # check that we have Internet access - bail quickly if we don't
-    HEAD=$(curl -s "https://auth.docker.io") || return $?
+    _=$(curl -s "https://auth.docker.io") || return $?
```

Comment:
```suggestion
    curl -Is "https://auth.docker.io" >/dev/null 2>&1 || return $?
```
to make `curl` perform a `HEAD` request and not spawn a subshell.